notus/scanner/messages/message.py

```diff
@@ -55,12 +55,26 @@ def _parse(cls, data: Dict[str, Union[int, str]]) -> Dict[str, Any]:
                 f"Invalid message type {message_type} for {cls.__name__}. "
                 f"Must be {cls.message_type}.",
             )
-        return {
-            "message_id": UUID(data.get("message_id")),
-            "group_id": UUID(data.get("group_id")),
-            "created": datetime.fromtimestamp(
+
+        try:
+            message_id = UUID(data.get("message_id"))
+        except (TypeError, ValueError) as e:
+            raise e.__class__(f"error while parsing 'message_id', {e}")
+        try:
+            group_id = UUID(data.get("group_id"))
```

Comment:
group_id is not a UUID

Reply:
Yes it is. At least we always took one and it is also documented as one

Reply:
It is not: https://github.com/greenbone/eulabeia/blob/main/docs/messaging.md but as discussed in the daily it is fine when it is a second pr.